packages/graph/src/graph2/graph.ts

```diff
@@ -432,16 +432,32 @@ export class Graph2 extends SVGZoomWidget {
     highlightVerticies(vertexMap?: { [id: string]: boolean }) {
         const context = this;
         const vertexElements = this._vertexG.selectAll<SVGGElement, VertexPlaceholder>(".graphVertex");
+        
+        const showLabel = context.showVertexLabels();
         vertexElements
             .classed("graphVertex-highlighted", d => !vertexMap || vertexMap[d.id])
             .style("filter", d => vertexMap && vertexMap[d.id] ? "url(#" + this.id() + "_glow)" : null)
-            .transition().duration(this.transitionDuration())
+            .transition().duration(context.showVertexLabelsOnHighlight() ? 0 : this.transitionDuration())
             .on("end", function (d) {
+                let _showLabel = showLabel;
                 if (vertexMap && vertexMap[d.id]) {
                     if (d.element.node() && d.element.node().parentNode) {
                         d.element.node().parentNode.appendChild(d.element.node());
                     }
+                    if(!_showLabel)_showLabel = context.showVertexLabelsOnHighlight();
```

Comment:
I will have a play, not sure if this should be needed?

Reply:
That line tells it to set showLabel to true if they have `showVertexLabelsOnHighlight` enabled